ui/src/message_popup/message_popup_page.jsx

```diff
@@ -20,7 +21,7 @@ function randomizedQuestionsWithStudents() {
   });
 }
 
-function logLocalStorage(record) {
+function logLocalStorage(type, record) {
```

Comment:
This doesn't look like it's using the `type` value anywhere here.  I can update that in a separate PR with the server-side changes too, I'll send that over next.